app/src/main/res/layout/list_item_offline_songs.xml

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
WARNING: [UnusedResources] The resource `R.layout.list_item_offline_songs` appears to be unused
 Unused resources make applications larger and slow down builds.